### ISSUE TITLE
Pull from nvidia/cuda failing

### ISSUE THREAD

**Issue by ra9hur** [NVIDIA/nvidia-docker#463]
Pull requests for cuda/cnn images are timeing out. Have tried,
- FROM nvidia/cuda:8.0-cudnn7-runtime-ubuntu16.04
- FROM nvidia/cuda:8.0-cudnn6-runtime-ubuntu16.04
- FROM nvidia/cuda:8.0-cudnn6-devel-ubuntu16.04
- FROM nvidia/cuda:8.0-cudnn6-runtime       (default ubuntu14.04)

Have consistently been getting the below error:
error pulling image configuration: Get https://dseasb33srnrn.cloudfront.net/registry-v2/docker/registry/v2/blobs/sha256/80/807549c5bf5e0a262364e93ed387fa4aa67ebbbc890773c3af3f47a916e284f5/data?Expires=1504965103&Signature=CWT0OFDpMys4HlIYHuyoMAwLiUyF~RcSKhKxm8jUzZEj8ZoHefrW8v1~6p4oJQvc8LOe8MRi7T9KgvSykhTIGhx4i6bDLDwhMPCITRO3BGtdNQaLeD1Azeccsl1CNphT601f3IIHvMkviwJn-2NU-n-QqlQnOz0bLtaG3dxRLG8_&Key-Pair-Id=APKAJECH5M7VWIS5YZ6Q: dial tcp 218.248.255.164:443: getsockopt: connection refused

To ensure that nothing is wrong with my laptop network configuration, have tried pulling other repositories from docker such as,
- FROM ros:kinetic-perception

Works fine.

**Comment by ra9hur** [comment]
Tried below commands:

$ sudo systemctl restart nvidia-docker
$ sudo systemctl stop nvidia-docker
$ sudo systemctl start nvidia-docker


$ journalctl -n -u nvidia-docker
-- Logs begin at Sun 2017-09-10 18:41:19 IST, end at Sun 2017-09-10 18:55:55 IST. --
Sep 10 18:55:39 raghu-Inspiron-3542 systemd[1]: Starting NVIDIA Docker plugin...
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:39 raghu-Inspiron-3542 systemd[1]: Started NVIDIA Docker plugin.
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:39 raghu-Inspiron-3542 nvidia-docker-plugin[3459]: /usr/bin/nvidia-docker-plu
Sep 10 18:55:55 raghu-Inspiron-3542 systemd[1]: Stopping NVIDIA Docker plugin...
Sep 10 18:55:55 raghu-Inspiron-3542 systemd[1]: Stopped NVIDIA Docker plugin.


$ curl -s localhost:3476/docker/cli
--volume-driver=nvidia-docker --volume=nvidia_driver_381.22:/usr/local/nvidia:ro --device=/dev/nvidiactl --device=/dev/nvidia-uvm --device=/dev/nvidia0

I upgraded driver from 375 to 381 and curl cmd is showing right version.

I had nvidia/cuda:8.0-cudnn5-runtime earlier. Just removed the container while uninstalling docker. Would it leave some traces behind ? Should it be cleaned before initiating pull request?
